src/System.Resources.ReaderWriter/src/project.lock.json

```diff
@@ -1,5 +1,5 @@
 {
-  "locked": true,
+  "locked": false,
```

Comment:
Is that the general guidance? I see it done differently all over the place. BTW, this file was autogenerated, i.e. I have not changed anything in it.